src/vendor/perf-goggles.js

```diff
@@ -11,19 +11,19 @@ export const signaturesUrl = (project = PROJECT) =>
 
 const dataPointsEndpointUrl = (project = PROJECT) =>
   `${TREEHERDER}/api/project/${project}/performance/data/`;
-const platformSuitesUrl = ({ frameworkId, platform, project }) =>
+const platformSuitesUrl = ({ framework, platform, project }) =>
```

Comment:
Do you have plans to upstream this vended module?
I would like to go back to the published library.
I can show you how to make local changes to perf-goggles while you work on Fx health.

Reply:
yes, I would like to upstream this.  I have a local fork of perf-goggles that synchronized with this file.